ResIndex.py

```diff
@@ -0,0 +1,23 @@
+from dataclasses import dataclass
+
+@dataclass
+class ResIndex:
+    df = ""
```

Comment:
is this a DataFrame or a string? the name suggest pd.DataFrame but you assign string value

Reply:
it's a string because it's the name of the dataframe, i'll change it to df_name or something more clear

Reply:
oh, ok, then `df_name` would be a good name :)